angularSTEP/src/app/current-profile-tab/current-profile-tab.component.ts

```diff
@@ -0,0 +1,96 @@
+import {AngularFirestore} from '@angular/fire/firestore';
+import {Component, OnInit, Input, NgZone} from '@angular/core';
+import {FormControl} from '@angular/forms';
+import {AngularFireAuth} from '@angular/fire/auth';
+import {User} from '../user';
+import {Converter} from '../converter';
+import {Recipe} from '../recipe';
+import {RecipeConverter} from '../recipe-converter';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-current-profile-tab',
+  templateUrl: './current-profile-tab.component.html',
+  styleUrls: ['./current-profile-tab.component.scss'],
+})
+export class CurrentProfileTabComponent implements OnInit {
+  @Input() userData!: User;
+  aboutMeForm: FormControl | null = null;
+  usersFollowing: User[] = [];
+  myRecipes: Recipe[] = [];
+
+  constructor(private afs: AngularFirestore, private fAuth: AngularFireAuth, private zone: NgZone, private router: Router) {}
+
+  ngOnInit() {
+    this.getFollowing();
+    this.getRecipes();
+  }
+
+  editValue(form: string) {
+    switch (form) {
```

Comment:
Nit: It its current form this looks like a job for an `if` statement

Reply:
Oops yes you are right. Originally I thought it would have more values to edit but realized now that this is not the case. Thanks!